### PR TITLE
metric logging  hotfix

### PR DESCRIPTION
**Pull request by trajan0x**
Update metrics exporter
[afef230](https://github.com/synapsecns/sanguine/pull/1161/commits/afef230001a6248903d9605412acc031f1ba659d) in prod

### CODE REVIEW

**Review by nautsimon — APPROVED** [review]

File: core/metrics/base.go [hunk 1]
```diff
@@ -123,10 +125,8 @@ func newBaseHandlerWithTracerProvider(buildInfo config.BuildInfo, tracerProvider
 	}
 
 	// TODO set up exporting the way we need here
-	metricExporter, err := stdout.New()
-	if err != nil {
-		return nil
-	}
+	metricExporter := noOpExporter{}
```
Comment (trajan0x) [inline core/metrics/base.go:128]: rename to noopmetricexporter

File: core/metrics/base.go [hunk 2]
```diff
@@ -142,3 +142,28 @@ func newBaseHandlerWithTracerProvider(buildInfo config.BuildInfo, tracerProvider
 }
 
 var _ Handler = &baseHandler{}
+
+type noOpExporter struct {
```
Comment (trajan0x) [inline core/metrics/base.go:146]: comment on what this does, add a constructor

File: core/metrics/base.go [hunk 3]
```diff
@@ -142,3 +142,37 @@ func newBaseHandlerWithTracerProvider(buildInfo config.BuildInfo, tracerProvider
 }
 
 var _ Handler = &baseHandler{}
+
+// noOpExporter is a no-op metric exporter that prevents any metrics from being exported.
+type noOpMetricExporter struct{}
+
```
Comment (nautsimon) [inline core/metrics/base.go:148]: comment

File: core/metrics/base.go [hunk 4]
```diff
@@ -142,3 +142,37 @@ func newBaseHandlerWithTracerProvider(buildInfo config.BuildInfo, tracerProvider
...
+func newNoOpExporter() noOpMetricExporter {
+	return noOpMetricExporter{}
+}
```
Comment (nautsimon) [inline core/metrics/base.go:151]: Constructor

File: core/metrics/base.go [hunk 5]
```diff
@@ -142,3 +142,37 @@ func newBaseHandlerWithTracerProvider(buildInfo config.BuildInfo, tracerProvider
 }
 
 var _ Handler = &baseHandler{}
+
+// noOpExporter is a no-op metric exporter that prevents any metrics from being exported.
+type noOpMetricExporter struct{}
+
+func newNoOpExporter() noOpMetricExporter {
```
Comment (trajan0x) [inline core/metrics/base.go:149]: return metric.Exporter

**Pull request merged**